### PR TITLE
Danish translation

### PR DESCRIPTION
**Pull request by naxoc**
We translated the page to Danish for use in our company. 

If you guys want to support multiple languages on the main site then here is a pull request. I didn't know what structure you would want, so I just made a folder with a new index.hml. Let me know if you need me to move stuff around :)

### CODE REVIEW

**Comment by mirzu** [comment]
Hi I'm sorry it has taken me so long to review this. I'm getting someone a little more familiar with Jekyll to review it. I'm almost certain this won't be an issue, but want to confirm that we wouldn't rather do this in the markdown.

**Comment by mirzu** [comment]
@rupl @elliotttf Can you review this and see how you feel about having the translation done in the rendered html?

File: da/index.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<!--[if IE 8]> <html class="no-js lt-ie9" lang="en" > <![endif]-->
```
Comment (rupl) [inline da/index.html:2]: `lang` needs to be switched from `en` here — easy to overlook since it's IE8 :)

**Comment by rupl** [comment]
Ultimately I think we'll need to switch templating around. However I don't want to block this awesome contribution on those grounds, so once we get that language attr switched out I'm :+1: with this!

**Comment by rupl** [comment]
Actually I'm just gonna clean this up myself, thanks for much for the translation, @naxoc!

**Comment by rupl** [comment]
:boom: this is live! http://consultancyscrum.org/da/

**Comment by naxoc** [comment]
Thanks @rupl !
When you switch templating around, just let me know if I can help with the Danish page :)

**Comment by rupl** [comment]
@naxoc I already took one pass at it, pull down latest `gh-pages` and take a look! I have also created issue #2 so we can surface your translation and perhaps inspire others to offer their own translations as well.

**Pull request merged**